tests/test_gossip.py

```diff
@@ -1034,7 +1037,10 @@ def test_gossip_addresses(node_factory, bitcoind):
     nodes = l2.rpc.listnodes(l1.info['id'])['nodes']
     assert len(nodes) == 1 and nodes[0]['addresses'] == [
         {'type': 'ipv4', 'address': '127.0.0.1', 'port': 2},
+        {'type': 'ipv4', 'address': '127.0.0.1', 'port': 19846},
         {'type': 'ipv6', 'address': '::', 'port': 3},
+        {'type': 'ipv6', 'address': '::', 'port': 19846},
+        {'type': 'torv3', 'address': 'vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion', 'port': 19846},
```

Comment:
In the case of Elements this test will fail.

We need somethings like if `19846 l1.info["network"] == "testnet" else 2xxx`